spinalcordtoolbox/scripts/sct_label_utils.py

```diff
@@ -104,8 +104,9 @@ def get_parser():
     func_group.add_argument(
         '-create-viewer',
         metavar=Metavar.list,
-        type=list_type(',', int),
-        help="Manually label from a GUI a list of labels IDs, separated with ','. Example: 2,3,4,5"
+        help="Manually label from a GUI a list of labels IDs. Provide a comma-separated list"
```

Comment:
```suggestion
        help="Manually label from a GUI a list of labels IDs. Provide a comma-separated list "
```

I'm sorry, there were 2 typos.